keps/sig-autoscaling/20181106-in-place-update-of-pod-resources.md

```diff
@@ -283,6 +292,16 @@ updates resource limit for the Pod and its Containers in the following manner:
 In all the above cases, Kubelet applies Container resource limit decreases
 before applying limit increases.
 
+#### Container resource limit update failure handling
+
+If multiple Containers in a Pod are being updated, and UpdateContainerResources
+CRI API fails for any of the containers, Kubelet will backoff and retry at a
```

Comment:
Can you clarify backoff?  Do you mean to say it will send an UpdateContainerResources to revert any containers which were successfully updated before the failing update?

Reply:
No. Chances are that revert operation of prior successful updates might also fail. If we hit this condition, we effectively pause and retry updating the specific container until success, and then proceed. I don't know if there's a better way.